pkg/controllers/dashboard/fleetcharts/controller.go

```diff
@@ -144,6 +144,15 @@ func (h *handler) onSetting(key string, setting *v3.Setting) (*v3.Setting, error
 		fleetChartValues["gitjob"] = gitjobChartValues
 	}
 
+	if extraValues, err := h.chartsConfig.GetChartValues(fleetconst.ChartName); err != nil {
+		// Missing extra config is okay, return the error otherwise
+		if !chart.IsNotFoundError(err) {
+			return nil, err
+		}
+	} else {
+		coalesceMap(fleetChartValues, extraValues)
```

Comment:
nit: 
we can simplify this without using an else
```suggestion
	extraValues, err := h.chartsConfig.GetChartValues(fleetconst.ChartName); 
	if err != nil && !chart.IsNotFoundError(err)  {
		// Missing extra config is okay, return the error otherwise
		return nil, err
	} 
	coalesceMap(fleetChartValues, extraValues)
```